libs/filabridge/include/filament/MaterialEnums.h

```diff
@@ -151,7 +151,7 @@ enum VertexAttribute : uint8_t {
 };
 
 static constexpr size_t MAX_LEGACY_MORPH_TARGETS = 4;
-static constexpr size_t MAX_MORPH_TARGETS = 128; // this is limited by filament::CONFIG_MAX_MORPH_TARGET_COUNT
+static constexpr size_t MAX_MORPH_TARGETS = 256; // this is limited by filament::CONFIG_MAX_MORPH_TARGET_COUNT
```

Comment:
the problem is that GLES3.0 doesn't guarantee more  than 256 layers in a 2D array, and with 256 morph targets we'd need 512 guaranteed.